compiler/testData/codegen/bytecodeListing/coroutines/spilling/booleanParameter.txt

```diff
@@ -104,3 +104,16 @@ public class helpers/EmptyContinuation {
     public method resumeWith(@org.jetbrains.annotations.NotNull p0: java.lang.Object): void
     public final inner class helpers/EmptyContinuation$Companion
 }
+
+@kotlin.Metadata
+public final class helpers/ResultContinuation {
```

Comment:
Should bytecode listing texts really be dumping that? Blackbox tests with CHECK_BYTECODE_LISTING don't.

Reply:
cc @ilmirus

Reply:
Yes, @pyos is right, this is a helper class and should not be dumped.

Reply:
Done.